.gitignore

```
@@ -1,5 +1,6 @@
 safetensors/target
-Cargo.lock
+safetensors/Cargo.lock
+safetensors/fuzz/Cargo.lock
```

Could we just combine them into one?
```suggestion
safetensors/**/Cargo.lock
```

---

Thanks !